Makefile

```
@@ -402,10 +402,14 @@ import-data: start-db
 
 .PHONY: import-borders
 import-borders: start-db-nowait
-	@$(assert_area_is_given)
-	# If CSV borders file already exists, use it without re-parsing
-	$(DOCKER_COMPOSE) run $(DC_OPTS) openmaptiles-tools sh -c \
-		'pgwait && import-borders $$([ -f "$(BORDERS_CSV_FILE)" ] && echo load $(BORDERS_CSV_FILE) || echo import $(PBF_FILE))'
+ifeq (,$(wildcard $(BORDERS_CSV_FILE)))
+        @$(assert_area_is_given)
+        @echo "Generating borders out of $(PBF_FILE)"
+else
+        @echo "Borders already exists. Useing $(BORDERS_CSV_FILE) to import borders"
+endif
+        $(DOCKER_COMPOSE) run $(DC_OPTS) openmaptiles-tools sh -c \
+                'pgwait && import-borders $$([ -f "$(BORDERS_CSV_FILE)" ] && echo load $(BORDERS_CSV_FILE) || echo import $(PBF_FILE))'
```

Simple switch from spaces to tabs

```suggestion
ifeq (,$(wildcard $(BORDERS_CSV_FILE)))
	@$(assert_area_is_given)
	@echo "Generating borders out of $(PBF_FILE)"
else
	@echo "Borders already exists. Useing $(BORDERS_CSV_FILE) to import borders"
endif
	$(DOCKER_COMPOSE) run $(DC_OPTS) openmaptiles-tools sh -c \
		'pgwait && import-borders $$([ -f "$(BORDERS_CSV_FILE)" ] && echo load $(BORDERS_CSV_FILE) || echo import $(PBF_FILE))'
```